### DISCUSSION TITLE
How to properly wrap existing nextjs app with electron?

### DISCUSSION THREAD

**Discussion by softmarshmallow** [vercel/next.js#19472] [Ideas]
When i say wrap, means some or alot modifications are ok to convert it as also-desktop-compatible application.

I Noticed and tried out nextron, which is it's own framework by itself, does not work well with existing-complex nextjs application.

Will be great to have an official example from nextjs, using below techs.

- nextjs
- electron
- monorepo
- micro frontend

wich all subdomains gets deployed separately, and also the electron side gets packed separately.

**Comment by jamesmosier** [comment]
There is an example currently for Next with Electron: https://github.com/vercel/next.js/tree/canary/examples/with-electron

But feel free to make an example including all the tech you included though!

**Reply by softmarshmallow** [reply]
I see. But that particular example is FOR electron, not WITH electron. which is not nextjs is built for. The example with separated web and app build configuration would make more sense. (I might be miss understanding :) )
